### ISSUE TITLE
Emails with addresses with name fail to send

### ISSUE THREAD

**Issue by mbi** [digitalvalley-nl/django-mailersend#3]
E.g. 
```python
send_mail(subject='Test subject', message='test body', from_email='Test sender <some@email.com>', recipient_list=['Some recipient <recipient@email.com>'])
```

This is perfectly okay according to [RFC 5322 ](https://www.rfc-editor.org/rfc/rfc5322) (page 45) or [section 3.4](https://www.rfc-editor.org/rfc/rfc5322#section-3.4) but  mailersend will respond with 

`HTTP 422 {"message":"The from.email must be a valid email address.","errors":{"from.email":["The from.email must be a valid email address."]}}`

The sent payload contains `{'email': 'Test sender <some@email.com>'}` while it should probably be parsed and split into 
```python
{
    "name": "Test sender",
    "email": "some@email.com",
}
```

**Comment by davidconde-nl** [comment]
Thanks for your report. I will investigate the issue.

**Comment by davidconde-nl** [comment]
Fixed in `v0.2.1-alpha`.

**Comment by mbi** [comment]
Thank you! :pray:
